docs/site-example/site.conf

```diff
@@ -209,8 +209,13 @@
     -- },
     -- define if the contact field is obligatory (optional)
     -- owner = {
-      -- obligatory = true
+      -- obligatory = true,
     -- },
+    -- Show/hide the password login
+    -- remote_login = {
+      -- show_password_form = true,
+      -- min_password_length = 12,
+    -- }
```

Comment:
same issue as in another comment: do we want to add a comma after the last entry?

Reply:
so this one is correct.

Reply:
no, there has to be a comma after the closing curly bracket to be consistent